e2e/node/src/node-server.test.ts

```diff
@@ -14,11 +14,16 @@ import {
 } from '@nx/e2e/utils';
 import { join } from 'path';
 
-xdescribe('Node Applications + webpack', () => {
+describe('Node Applications + webpack', () => {
   let proj: string;
   beforeAll(() => {
     proj = newProject({
       packages: ['@nx/node'],
+      // npm has resolution for ajv some packages require ajv6 and some require ajv8 and npm resolves it to ajv6 (Error: Cannot find module 'ajv/dist/compile/codegen')
```

Comment:
We should at least add a //TODO to re-enable the tests for other package managers.

@jaysoo Do you know how we can either align these versions or mitigate this issue with the default options of npm?